### PR TITLE
7.x 2.x spotlights

### PR DESCRIPTION
**Pull request by cjwest**
# READY FOR REVIEW

# Summary
- for [SOE-2502](https://stanfordits.atlassian.net/browse/SOE-2502)
This updates the view so that the quote marks don't display when there's no quote

- for [SOE-2728](https://stanfordits.atlassian.net/browse/SOE-2728)
This removes the link on the title on the spotlight  node page

- for [SOE-1929](https://stanfordits.atlassian.net/browse/SOE-1929)
This adds the Views Display for related Spotlights

- for  [SOE-3100](https://stanfordits.atlassian.net/browse/SOE-3100)
This adds the "Alt" tag to the spotlight images.

# Needed By (Date)
- Chameleon release

# Criticality
- Client is eagerly awaiting this.

# Steps to Test
- Switch to this branch
git checkout 7.x-2.x-spotlights
drush rr
drush fr stanford_people_spotlight_views
drush fr stanford_people_spotlight (added by @minorwm)
drush cc all

## For quotes:  [SOE-2502](https://stanfordits.atlassian.net/browse/SOE-2502)
- Create a context at `/admin/structure/context`
- For each of these views place a block associated with that view or the views display

1. Stanford People Spotlight: 1, Vertical, Span4, Card
2. Stanford People Spotlight: 1, Vertical, Span6, Card
3. Stanford People Spotlight: 2, Vertical, Span6, Card
4. Stanford People Spotlight: 3, Vertical, Card
5. and Stanford People Spotlight: 3, Vertical, Card: Attachment First Spot details
6. Stanford People Spotlight: FW Banner - Quote
7. and Stanford People Spotlight: FW Banner - Quote: Block - quote, not random
8. Stanford People Spotlight: Horizontal Card

- Verify that each displays as expected
- Then delete the quote and verify that no quote marks appear. _Note, this may be tricky because these are random views and when you refresh, you'll see a different node. Have fun!_

screen shot 2018-07-11 at 18 06 16

## For spotlight nodes [SOE-2728](https://stanfordits.atlassian.net/browse/SOE-2728)
Click on a spotlight on any page to navigate to the spotlight node page
Verify that the title no longer has a link

screen shot 2018-07-11 at 18 08 47

## For related spotlights [SOE-1929](https://stanfordits.atlassian.net/browse/SOE-1929)
Navigate to a Spotlight node page, scroll to the bottom, and verify the "Related Spotlights" appears.

screen shot 2018-07-13 at 11 50 40

## For  [SOE-3100](https://stanfordits.atlassian.net/browse/SOE-3100)
Edit a spotlight
Verify "Alternative Text" is in fact there!

screen shot 2018-07-13 at 15 20 48

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
### Implementation tickets
[SOE-2728](https://stanfordits.atlassian.net/browse/SOE-2728)
[SOE-2502](https://stanfordits.atlassian.net/browse/SOE-2502)
[SOE-1929](https://stanfordits.atlassian.net/browse/SOE-1929)
[SOE-3100](https://stanfordits.atlassian.net/browse/SOE-3100)

### PR ticket
https://stanfordits.atlassian.net/browse/SOE-3258

## Folks to notify
@kerri-augenstein 
@annawatt 
# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Review by minorwm — APPROVED** [review]

**Comment by minorwm** [comment]
Everything looked good! Merging.

**Comment by sherakama** [comment]
@cjwest @minorwm you both should be ignoring file permission changes in your git repositories

See:
https://github.com/Gizra/KnowledgeBase/wiki/How-to-set-Git-to-ignore-chmod-changes

File: modules/stanford_people_spotlight_views/stanford_people_spotlight_views.views_default.inc [hunk 1]
```diff
@@ -484,10 +509,8 @@ function stanford_people_spotlight_views_views_default_views() {
       <div class="spotlight-department">
         <p>[field_s_ppl_spot_department]</p>
       </div>
-      <div class="spotlight-quote">
-        <p>[field_s_ppl_spot_quote]</p>
-      </div>
-      <div class="edit-link">[edit_node]</div>
+      [field_s_ppl_spot_quote_1]
+       <div class="edit-link">[edit_node]</div>
```
Comment (sherakama) [inline modules/stanford_people_spotlight_views/stanford_people_spotlight_views.views_default.inc:513]: The edit node link should contain the full markup so when a user does not have permission to edit the item they are not given an empty div. Please wrap the `<div>` in to `[edit_node]`

Comment (sherakama) [reply modules/stanford_people_spotlight_views/stanford_people_spotlight_views.views_default.inc:513]: @cjwest

**Pull request merged**